spring-boot/config-server/src/main/java/eu/coatrack/admin/server/service/filter/CheckIfUrlAndCredentialsFitWithEachOtherFilter.java

```diff
@@ -16,15 +16,15 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
-import org.springframework.web.filter.GenericFilterBean;
+import org.springframework.web.filter.OncePerRequestFilter;
 
 /**
  *
  * @author perezdf // TODO Consider the idea to split up the filter in case, it
  * keep growing up
  */
 @Component
-public class CheckIfUrlAndCredentialsFitWithEachOtherFilter extends GenericFilterBean {
+public class CheckIfUrlAndCredentialsFitWithEachOtherFilter extends OncePerRequestFilter {
```

Comment:
Maybe something like ```UrlAndCredentialsConsistencyFilter``` would be better name.

Reply:
Done